### DISCUSSION TITLE
wordpress-activitypub with friendica?

### DISCUSSION THREAD

**Discussion by Mannshoch** [Automattic/wordpress-activitypub#208] [General]
I have a Wordpress which I use with this addon an will get a better integration with https://github.com/pfefferle/wordpress-activitypub/issues/1
Recently I found out that [friendica](https://github.com/friendica/friendica) also work with php and so I'm able to use it with a common webserver.
In friendica there is an Addon to post to wordpress.

I'm not that experienced. But It seems that Wordpress and Friendica could be a nice couple. Do you see that we could use friendica to follow specific authors and Wordpress-ActivityPub to follow the whole Wordpressblog?

Has anyone additional input? Some proposals, warnings or other Input?
Would you see that a migration to that with all old post will work?

**Comment by pfefferle** [comment]
I am not sure I understand your point completely. What I can say is, that WordPress is already compatible with Friendi.ca using the ActivityPub plugin.
